Sources/ExtensionDictionary.swift

```diff
@@ -24,8 +24,50 @@
 //
 
 import Foundation
-
-internal extension Dictionary {
+public extension Dictionary {
```

Comment:
i don't get that warning. but if you do - okay.

then we should explicitly make `init(elements:)` and `flatMap(_:)` as `internal`